controllers/cloud.redhat.com/makers/maker.go

```diff
@@ -169,12 +169,7 @@ func (m *Maker) makeService(pod crd.PodSpec) error {
 		ports = append(ports, webPort)
 	}
 
-	labels := m.App.GetLabels()
-	labels["pod"] = nn.Name
-	m.App.SetObjectMeta(&s, crd.Name(pod.Name), crd.Labels(labels))
-
-	s.Spec.Selector = labels
-	s.Spec.Ports = ports
+	utils.MakeService(&s, nn, map[string]string{"pod": nn.Name}, ports, m.App)
```

Comment:
You missed using your fancy type def :)

Reply:
makers doesn't have it and for this single use - probably not worth it